src/main/java/org/apereo/openequella/adminconsole/util/ExecUtils.java

```diff
@@ -18,280 +18,39 @@
 package org.apereo.openequella.adminconsole.util;
 
 import java.io.File;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.URL;
-import java.net.URLDecoder;
 import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.StringTokenizer;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class ExecUtils {
-  public static final String PLATFORM_SOLARIS_X86 = "solaris-x86";
-  public static final String PLATFORM_SOLARIS_SPARC = "solaris-sparc";
-  public static final String PLATFORM_SOLARIS64 = "solaris64";
-  public static final String PLATFORM_LINUX = "linux";
-  public static final String PLATFORM_LINUX64 = "linux64";
-  public static final String PLATFORM_WIN = "win";
-  public static final String PLATFORM_WIN32 = "win32";
-  public static final String PLATFORM_WIN64 = "win64";
-  public static final String PLATFORM_MAC = "mac";
-
   private static final Logger LOGGER = LoggerFactory.getLogger(ExecUtils.class);
-  private static final String[] EXE_TYPES = {
-    "", ".exe", ".bat"
-  };
-
-  public static File findExe(File file) {
-    return findExe(file.getParentFile(), file.getName());
-  }
-
-  public static File findExe(File path, String exe) {
-    for (String exeType : EXE_TYPES) {
-      File exeFile = new File(path, exe + exeType);
-      if (exeFile.exists() && exeFile.canExecute()) {
-        return exeFile;
-      }
-    }
-    return null;
-  }
-
-  public static ExecResult exec(List<String> options) {
-    return exec(options.toArray(new String[options.size()]));
-  }
-
-  public static ExecResult exec(String... command) {
-    return exec(command, null, null);
-  }
 
-  public static ExecResult exec(String command, String[] env, File dir) {
-    Map<String, String> envMap = new HashMap<String, String>();
-    for (String e : env) {
-      StringTokenizer st2 = new StringTokenizer(e, "=");
-      if (st2.hasMoreTokens()) {
-        String key = st2.nextToken();
-        if (st2.hasMoreTokens()) {
-          envMap.put(key, st2.nextToken());
-        }
-      }
-    }
-    return exec(splitCommand(command), envMap, dir);
-  }
-
-  public static ExecResult exec(String[] cmdarray, Map<String, String> additionalEnv, File dir) {
+  public static int exec(String[] cmdarray, 
+      Map<String, String> additionalEnv, 
+      File dir) {
```

Comment:
Not a big deal : what about putting these three parameters back in one line?

Reply:
Done